UPGRADING.md

```diff
@@ -6,6 +6,15 @@
 
 The legacy JSON-WP protocol drivers (using Selenium v3 and Appium v11) are no longer supported. 
 
+### Removed steps
+
+| Removed step  | Replacement                     |
+|---|---------------------------------|
+| `I wait for {int} span(s)`  | I wait to receive {int} span(s) |
```

Comment:
I don't see this in this PR - is that already made?

Reply:
Yes, basically.  `I wait to receive {int} span(s)` spans is already in the repo and the change here is just to tidy away the one we don't want (in the deleted deprecated_steps.rb file).

Reply:
I later realised that that's not the right step to direct people to, as the deprecated one wait for _at least_ the number of spans.  I'll correct that.